### PR TITLE
Feat/view quests to claim rewards

### LINKED ISSUE #595: View quests to claim rewards
Loom : https://www.loom.com/share/7c8bcf7f8e36417683e68599caecb495

Figma : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/%F0%9F%9A%80-Starknet-Quest?type=design&node-id=4310%3A5471&mode=design&t=8bKmyIjFe5zHvCIr-1

technical specifications:
- using the endpoint - `/boost/get_pending_claims?addr=${addr}`
This will show all rewards which can be claimed by the user from the boosts section

### PR DESCRIPTION
**Pull request by Ayoazeez26**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Feature
- Testing

Resolves: #595

### CODE REVIEW

**Comment by ayushtom** [comment]
Initial review - 
- dont show to claim tab if wallet is not connected
- navigate to `/quest-boost/5` which is the boost page for quest id 23. If i click on claim reward or the card too.

**Comment by Ayoazeez26** [comment]
> * navigate to `/quest-boost/5` which is the boost page for quest id 23. If i click on claim reward or the card too.

@ayushtom I believe this is the current implemented behaviour, correct me if I'm wrong.

File: utils/quest.ts [hunk 1]
```diff
@@ -18,3 +18,26 @@ export function pickRandomObjectsFn({
     return questArray;
   }
 }
+
+export const getClaimableQuests = (
+  quests: QuestDocument[],
+  pendingBoostClaims: PendingBoostClaim[] | undefined
+) :ClaimableQuestDocument[] | undefined => {
+  if (pendingBoostClaims && pendingBoostClaims.length) {
```
Comment (ayushtom) [inline utils/quest.ts]: can write this as 
if(!pendingBoostClaims) return;
// rest of the code

helps with avoiding nested standalone if blocks

**Review by ayushtom — CHANGES_REQUESTED** [review]

**Review by ayushtom — APPROVED** [review]
lgtm

**Pull request merged**